spec/calculate_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "calculator.rb"
+
+describe Calculator do
+  
+  describe " #current_basket" do
+    it "should respond to two arguments" do
+      expect(Calculator).to respond_to(:current_basket).with(2).argument
+    end
+
+    it "should return a hash with the item_name, quantity and subtotal" do
+      order = {"Pilau rice": 1}
+      dishes = {"Pilau rice": 3.00}
+      expect(Calculator.current_basket(order, dishes)).to eq ([{item_name: "Pilau rice", quantity: 1, subtotal: 3.0}])
+    end
+
+    it "should return a hash with the item_name, quantity and subtotal for MULTIPLE items" do
```

Comment:
Maybe you could add a test for three dishes ordered? Or a test for an edge case where no dishes ordered